### ISSUE TITLE
Too much Padding due to Pandora changes

### ISSUE THREAD

**Issue by jnjosh** [jnjosh/PandoraBoy#2]
When running PandoraBoy, the UI should fill the view without showing the background area...

![Not Filling View](http://f.cl.ly/items/2C0I0L0E3f2C3W3U2z1n/Screen%20Shot%202011-08-03%20at%201.20.23%20AM.png)

**Comment by srogers** [comment]
Yeah - I just had to reload the Pandora player to get the latest updates and there were already scroll bars because of the ads, but now with the latest changes the controls and playing music won't fit anymore. They've changed the shape of the music icons and just generally revamped the whole player.
